GSoC-2025/pranav_kapoor.md

```diff
@@ -0,0 +1,80 @@
+## Personal details
+- **Full name:** Pranav Kapoor  
+- **Email:** pranav33317@gmail.com  
+- **GitHub username:** pranav33317  
+- **Zulip username:** Pranav Kapoor  
+- **Location & time-zone:** New Delhi, India (IST, GMT+5:30)  
+- **Personal website / project portfolio:** [www.github.com/pranav33317](https://www.github.com/pranav33317)  
+- **Code contribution:**  
+  - PR: [https://github.com/brainglobe/brainglobe-atlasapi/pull/546#pullrequestreview-2729541724](https://github.com/brainglobe/brainglobe-atlasapi/pull/546#pullrequestreview-2729541724)  
+    I tried to disable parallel mesh creation in various files to simplify mesh creation while simultaneously working on [Issue #429](https://github.com/brainglobe/brainglobeatlasapi/issues/429).  
+- **Proposal discussion link:**  
+  - [https://github.com/brainglobe/brainglobe-atlasapi/pull/554](https://github.com/brainglobe/brainglobe-atlasapi/pull/554)  
```

Comment:
This should be a link to the discussion about your proposed project, i.e. this PR.